cogs/eightball.py

```diff
@@ -31,7 +32,10 @@ async def eightball(self, ctx, *, question):
                     'Very doubtful',
                     'No']
 
-        await ctx.send(f"Question: {question}\nAnswer: {random.choice(responses)}")
+        message = discord.Embed(title="8 Ball")
```

Comment:
add a `colour= discord.Colour.orange()` after the title

Reply:
Okay. Just gonna take a minute